Update micro-bookmark to v2.5.0

haqk opened the pull request:
Updates the `bookmark` plugin manifest to **v2.5.0**.

- Single latest `Versions` entry pointing at the repo tag archive (live now): https://github.com/haqk/micro-bookmark/archive/v2.5.0.zip
- Requires `micro >= 2.0.0`.

The channel currently ships an old build, so `micro -plugin install bookmark` is several releases behind. v2.5.0 adds content-anchored persistence (bookmarks survive edits made while a file is closed), fixes persistence on quit and for command-line-opened files, and adds select-between-bookmarks commands. Changelog: https://github.com/haqk/micro-bookmark/blob/main/CHANGELOG.md

Maintained at https://github.com/haqk/micro-bookmark

samdmarshall committed:
adding fzf plugin

zyedidia committed:
Merge pull request #3 from samdmarshall/master

adding fzf plugin

Commit:
Add editorconfig repo.json to channel.json

Commit:
Update README for editorconfig

Theodus committed:
add Pony plugin to channel

Theodus committed:
add Pony plugin to README

zyedidia committed:
Merge pull request #5 from Theodus/master

Add Pony plugin

zyedidia committed:
Merge branch 'editorconfig' of https://github.com/10sr/plugin-channel into 10sr-editorconfig

zyedidia committed:
Merge branch '10sr-editorconfig'

zyedidia committed:
Fix readme

Commit:
Added crystal plugin

Commit:
Readme

zyedidia committed:
Merge pull request #7 from ColinRioux/master

Added crystal plugin

Commit:
Add gotham colorscheme

Commit:
Update README

Commit:
Align grid in README

Commit:
Add misspell plugin

zyedidia committed:
Merge pull request #8 from november-eleven/master

Gotham colorscheme

Theodus committed:
add monokai-dark colorscheme

zyedidia committed:
Merge branch 'master' into master

zyedidia committed:
Merge pull request #9 from onodera-punpun/master

Add misspell plugin

Theodus committed:
resolve conflicts

zyedidia committed:
Merge branch 'master' into master

zyedidia committed:
Merge pull request #10 from Theodus/master

add monokai-dark colorscheme

dmage committed:
Fix syntax error

zyedidia committed:
Merge pull request #11 from dmage/patch-1

Fix syntax error

samdmarshall committed:
adding scratch plugin

Commit:
docs: Update README

Commit:
chore: Update URL

zyedidia committed:
Merge pull request #14 from novln/master

Update Gotham URL

NicolaiSoeborg committed:
Update channel.json

Fixes https://github.com/zyedidia/micro/issues/526

NicolaiSoeborg committed:
... also update desc

zyedidia committed:
Merge pull request #15 from NicolaiSoeborg/patch-2

Add (text) manipulator plugin

NicolaiSoeborg committed:
Merge remote-tracking branch 'refs/remotes/micro-editor/master'

NicolaiSoeborg committed:
Add filemanager plugin

samdmarshall committed:
Merge branch 'master' into scratch

zyedidia committed:
Merge pull request #16 from NicolaiSoeborg/filemanager

Add a filemanager plugin

zyedidia committed:
Merge branch 'master' into scratch

zyedidia committed:
Merge pull request #12 from samdmarshall/scratch

adding scratch plugin

nalanj committed:
Add rubocop plugin to repo.json

nalanj committed:
Add rubocop to readme

zyedidia committed:
Merge pull request #17 from computerers/master

Adds rubocop plugin

zyedidia committed:
Remove filemanager plugin

The filemanager plugin has bugs which need to be resolved before it can
be an official plugin.

dermetfan committed:
add VCS plugin

zyedidia committed:
Merge pull request #18 from dermetfan/vcs-plugin

add VCS plugin

zyedidia committed:
Remove outdated colorschemes

zyedidia committed:
Revert "Remove outdated colorschemes"

This reverts commit 5eaff66950a650b6ed777299907111fce9460fd7.

NicolaiSoeborg committed:
Revert 81999d3: remove filemanager plugin

zyedidia committed:
Merge pull request #20 from NicolaiSoeborg/master

Revert 81999d3: remove filemanager plugin

sum01 committed:
Add fmt plugin

zyedidia committed:
Merge pull request #22 from sum01/add_fmt

Add fmt plugin

sum01 committed:
Fix the weird URL on fmt
It was breaking the website's plugin fetchData call.
Ref https://github.com/micro-editor/micro-editor.github.io/issues/6

zyedidia committed:
Merge pull request #25 from sum01/fmt-fix

Fix the weird URL on fmt

tommyshem committed:
changed snippets location

tommyshem committed:
change snippets url

project is moving to new repo

zyedidia committed:
Merge pull request #28 from tommyshem/master

change url for snippets plugin

BurnerWah committed:
Removed Rubocop plugin

zyedidia committed:
Merge pull request #37 from YaBoiBurner/master

Remove Rubocop plugin

Lisiadito committed:
add joinLine plugin

zyedidia committed:
Merge pull request #39 from Lisiadito/master

add joinLine plugin

Lisiadito committed:
fix broken update mechanism due to missing comma

zyedidia committed:
Merge pull request #40 from Lisiadito/master

[URGENT] fix broken plugin mechanism due to missing comma

Commit:
pin editorconfig repo.json to sha with a pinned channel.json

zyedidia committed:
Merge pull request #46 from joshuarli/feat/pin-editorconfig-to-micro-1-compat

pin editorconfig repo.json to sha with a pinned channel.json

zyedidia committed:
Update link to go plugin repo.json

zyedidia committed:
Update channel.json

zyedidia committed:
Update plugin links to point to updated versions

Commit:
restore upstream master repo.json for editorconfig

zyedidia committed:
Merge pull request #47 from joshuarli/patch-1

chore: restore upstream master repo.json for editorconfig

sparques committed:
Add quoter and bounce plugin (#48)

* Add quoter plugin

* Add bounce plugin

* Update README.md to include quoter and bounce

sparques committed:
put in the right branch/tag for quoter and bounce (#49)

because I didn't do it right the first time.

squeek502 committed:
Add zigfmt plugin (#51)

priner committed:
Add aspell plugin (#52)

Lisiadito committed:
update channel.json to point back to original updated repo (#54)

zyedidia committed:
Update wc plugin url

zyedidia committed:
Point wc plugin back to updated-plugins repo

masflam committed:
add jlabbrev plugin (#56)

gandarez committed:
Add wakatime plugin

zyedidia committed:
Add wakatime plugin (#57)

zyedidia committed:
Update channel.json

KiranWells committed:
Add nord-tc colorscheme link

KiranWells committed:
Add nord-tc

zyedidia committed:
merge

zyedidia committed:
Merge branch 'KiranWells-master'

KiranWells committed:
Update the Nord colors name (#60)

KiranWells committed:
Update Nord Colors name again due to a bug (#62)

* Remove the "-" in nord colors to fit the new name.

* Update the name in the comment while I am at it.

a11ce committed:
Add YAPF plugin (#61)

zyedidia committed:
Update channel.json

Commit:
Add quickfix plugin

haqk committed:
Add bookmark plugin (#63)

Co-authored-by: HaQK <i@haqk.cc>

zyedidia committed:
Merge branch 'master' of https://github.com/serge-v/plugin-channel into serge-v-master

zyedidia committed:
Merge branch 'serge-v-master'

dmaluka committed:
Add detectindent plugin (#67)

terokarvinen committed:
Add jump plugin (#66)

* Add jump plugin

* Fix URL in channel.json

Co-authored-by: Tero Karvinen <teropkg@gmx.com>

zyedidia committed:
Update channel.json

zyedidia committed:
Update snippets link

AndCake committed:
:star2: adds LSP plugin (#85)

zyedidia committed:
Make go plugin internal to plugin channel

zyedidia committed:
Make all plugins internally controlled

zyedidia committed:
Remove comment plugin (provided by default)

zyedidia committed:
Forgot editorconfig

zyedidia committed:
Add missing websites and most licenses

zyedidia committed:
Update readme

zyedidia committed:
Fixes

zyedidia committed:
Add PR template

haqk committed:
Update micro-bookmark.json

New release

taconi commented:
Hello, your `repo.json` does not have the and `License` field, could you add this metadata?

haqk committed:
Update micro-bookmark to v2.3.10

Bumps version, switches Url to the plugin-channel releases convention,
and adds License field.

haqk committed:
Update micro-bookmark to v2.3.11

haqk committed:
Update micro-bookmark to v2.5.0
